native/src/constants/contentDirection.ts

```diff
@@ -9,3 +9,15 @@ export const contentDirection = (contentLanguage: string): 'row' | 'row-reverse'
   isContentDirectionReversalRequired(contentLanguage) ? 'row-reverse' : 'row'
 export const contentAlignment = (contentLanguage: string): 'right' | 'left' =>
   isContentDirectionReversalRequired(contentLanguage) ? 'right' : 'left'
+export const isRTLText = (text: string): boolean => {
```

Comment:
Could you please add some tests for that?

Reply:
check last commit